src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -949,7 +949,8 @@ static ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
 
     md->log_max_dci_stream_channels = UCT_IB_MLX5DV_GET(cmd_hca_cap, cap,
                                                         log_max_dci_stream_channels);
-    ucs_assert(md->log_max_dci_stream_channels < 8);
+    md->log_max_dci_stream_channels = ucs_min(md->log_max_dci_stream_channels,
+                                              8);
```

Comment:
why 8? should we define some constant?

Reply:
> why 8? should we define some constant?

added `UCT_IB_MLX5_MD_MAX_DCI_CHANNELS` constant